vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -345,11 +383,14 @@ void WinRTWebSocketResource::SendBinary(const string& base64String)
 
 void WinRTWebSocketResource::Close(CloseCode code, const string& reason)
 {
+  if (m_readyState == ReadyState::Closing || m_readyState == ReadyState::Closed)
```

Comment:
>if [](start = 2, length = 2)

Is this going to be accessed from single thread, or this api is multithreaded?

Reply:
It can potentially be accessed from multiple threads.